config/settings/test.yml

```diff
@@ -69,8 +69,13 @@ vba_documents:
 
 bgs:
   mock_response_location: /cache/bgs
-  mock_responses: true
+  mock_responses: false
```

Comment:
Shouldn't this be true in test? or is this to record a cassette?

Reply:
Mock false allows VCR to work. Mock true would be like Betamocks being on.